EnvironmentSetup/AWS/Source/Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV TF_DATA_DIR=/terraform-state
 RUN apt-get update && apt-get install awscli software-properties-common apt-transport-https locales curl tar git procps libncurses-dev -y && \
 	locale-gen en_US.utf8 && locale -a  && \
 	update-locale LANG=en_US.UTF-8 LANG=en_US.UTF8 LC_CTYPE=en_US.UTF8 LC_COLLATE=en_US.UTF8 && \
-	curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" && \
+	curl -LO "https://dl.k8s.io/release/v1.23.0/bin/linux/amd64/kubectl" && \
```

Comment:
Is there any way to show this more conspicuously in a configuration variable? At some point in the future we may encounter the inverse of this bug where our kubectl version is too far out of date and someone will have to go digging for this line of code to fix that as things stand in this PR.

Reply:
Sure, I updated the Dockerfile.
https://github.com/WolframResearch/WAS-Kubernetes/blob/f33b90bba939a9691315dd46bb229e16f02f6f20/EnvironmentSetup/AWS/Source/Dockerfile#L2

Reply:
> Sure, I updated the Dockerfile.
> 
> https://github.com/WolframResearch/WAS-Kubernetes/blob/f33b90bba939a9691315dd46bb229e16f02f6f20/EnvironmentSetup/AWS/Source/Dockerfile#L2



> Sure, I updated the Dockerfile.
> 
> https://github.com/WolframResearch/WAS-Kubernetes/blob/f33b90bba939a9691315dd46bb229e16f02f6f20/EnvironmentSetup/AWS/Source/Dockerfile#L2

This looks much more readable.